golem_cli/src/command/provider.rs

```diff
@@ -302,6 +302,36 @@ impl YaProviderCommand {
             .spawn()?)
     }
 
+    pub async fn set_cert_rule(self, cert: &str, rule: &str) -> anyhow::Result<()> {
+        let mut cmd = self.cmd;
+
+        let output = cmd
+            .args([
+                "rule",
+                "set",
+                "outbound",
+                rule,
+                "import-cert",
+                cert,
+                "--mode",
+                "all",
+            ])
+            .stderr(Stdio::piped())
+            .stdout(Stdio::null())
+            .stdin(Stdio::null())
+            .output()
+            .await
+            .context("failed setting partner rule")?;
+
+        if output.status.success() {
+            Ok(())
+        } else {
+            let output = String::from_utf8_lossy(&output.stderr);
+            Err(anyhow::anyhow!("{}", output))
+        }
+    }
+
+    #[allow(dead_code)]
     pub async fn add_certs<I, S>(self, certs: I) -> anyhow::Result<()>
```

Comment:
If this function is not used is it really worth keeping it here?

Reply:
leftover